tests/integration/sentinel/schedules/outbound.spec.js

```diff
@@ -2,6 +2,9 @@ const utils = require('@utils');
 const sentinelUtils = require('@utils/sentinel');
 const chai = require('chai');
 
+const sentinelDate = utils.getSentinelDate();
```

Comment:
I think the issue you have is that you are calling this function here, when the file is loaded, before containers are up. Can you try moving this in the beforeEach or before ?